### PR TITLE
Add component to display how to install config

### PR DESCRIPTION
**Pull request by letieu**
🤔 I think it would be even better if I could see how to clone and try the Neovim config on this page, so I'll add it myself! 😎

If you agree, please merge it into the main branch. 🤝

Screenshot 2023-04-12 at 11 30 53 PM

### CODE REVIEW

**Comment by yutkat** [comment]
I don't think this installation method is appropriate because the location of Neovim configuration file in dotfiles is different for everyone.

**Comment by codicocodes** [comment]
> I don't think this installation method is appropriate because the location of Neovim configuration file in dotfiles is different for everyone.

Good point. If the script takes into account the root it might work though, I will experiment with it a bit.

I like the idea.

**Comment by codicocodes** [comment]
I accidentally closed this btw when I removed the dev branch, can't find a way to reopen it.

File: src/lib/components/NeovimConfigInstall.svelte [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<script>
+	import { faCopy } from '@fortawesome/free-solid-svg-icons';
+	import Fa from 'svelte-fa';
+
+  export let owner;
+  export let repo;
+
+  const code = `
+git clone https://github.com/${owner}/${repo}.git ~/.config/${owner}/${repo}
+NVIM_APPNAME=~/.config/${owner}/${repo} nvim`;
```
Comment (codicocodes) [inline src/lib/components/NeovimConfigInstall.svelte:10]: We need to use `NVIM_APPNAME=~/.config/${owner}/${repo}/${root} nvim` I believe in order to get to the nvim directory if they use a mono repo

**Pull request closed without merging**